src/psyclone/tests/nemo/transformations/openacc/kernels_directive_test.py

```diff
@@ -56,20 +56,6 @@
                  "end program do_loop\n")
 
 
```

Comment:
pylint complains that `InternalError` is imported but not used.

Reply:
Removed.